### ISSUE TITLE
'Enable docstring style linting' setting not saved in Preferences

### ISSUE THREAD

**Issue by rlabbe** [spyder-ide/spyder#10139]
### What steps will reproduce the problem?

My environement has Enable docstring style linting enabled, so the followings steps reproduce my problem.

1. Open Preferences
2. go to docstring style tab in Completion and linting
3. Uncheck 'Enable docstring style linting
4. Press OK to close Preferences, saving your change
5. Open Preferences, return to dosctring style tab. The checkbox is checked, whereas it should be unchecked due to step 3.

## Versions

* Spyder version: 4.0.0b4 
* Python version: 3.7.3
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 7

### Dependencies

```
pygments >=2.0    :  2.4.2 (OK)
qdarkstyle >=2.6.4:  2.7 (OK)
sphinx >=0.6.6    :  2.1.2 (OK)
pyls >=0.27.0     :  0.28.2 (OK)
nbconvert >=4.0   :  5.5.0 (OK)
pandas >=0.13.1   :  0.25.1 (OK)
numpy >=1.7       :  1.16.4 (OK)
sympy >=0.7.3     :  1.4 (OK)
cython >=0.21     :  0.29.13 (OK)
qtconsole >=4.5.2 :  4.5.4 (OK)
IPython >=4.0     :  7.8.0 (OK)
matplotlib >=2.0.0:  3.1.1 (OK)
pylint >=0.25     :  2.3.1 (OK)
```

**Comment by ccordoba12** [comment]
@dalthviz, can you reproduce this on Windows?

**Comment by ccordoba12** [comment]
@rlabbe, I had to impose a temporary ban to all people that have not contributed to this repo to prevent you opening more bug reports for today.

I appreciate your enthusiasm but your high amount of issues puts a lot of stress to our little team, trying to verify, reproduce, discuss and solve them.

Please try to limit your number of issues to a reasonable amount (3 per day) from now on, to give us time to manage them.

Thanks for understanding!

**Comment by goanpeca** [comment]
I can reproduce this.

**Comment by dalthviz** [comment]
@ccordoba12 @goanpeca yep on Windows happens for me too

**Comment by ccordoba12** [comment]
Yeah, me too, but I don't understand when this problem appeared.

Thanks for taking a look at it @goanpeca!

**Comment by goanpeca** [comment]
>Yeah, me too, but I don't understand when this problem appeared.

Yeah me neither, feels like we had solved this at some point in the past, but it keeps appearing :-| 

It has to do with how the signal for the checkbox is emitted (with an extra bool or not)

**Comment by goanpeca** [comment]
Ok, it was not what I thought but I found the issue! :-)

**Comment by goanpeca** [comment]
I have fixed the issue on PR #10145

But this fixes the UI problems, however, the actual action is not working and I opened https://github.com/spyder-ide/spyder/issues/10144 to track it

**Comment by ccordoba12** [comment]
> Please try to limit your number of issues to a reasonable amount (3 per day) from now on, to give us time to manage them.

@rlabbe, after talking with @goanpeca, we decided that we'll able to handle up to 10 issues per day from you, in case you want to keep opening them.

Sorry for the temporary ban (I already disabled it, which means everything is back to normal again). I hope you'd understood our reasons for doing it.

**Comment by rlabbe** [comment]
@ccordoba12 I don't see the value in debating your work flow choices, but I found it quite odd. The bugs exist, you should want to know about them, and I'm not going to remember to submit something in N days because I reached some arbitrary limit. I had assumed you would deal with them as you had time (days or weeks later) since this is beta software, not strive to respond as quickly as you have! But I'll certainly abide by your rules.

For what it's worth I'm using 4.0.0b4 at work, to do real work. These bugs affect my productivity, as does reporting them, and then dealing with all the emails that show up in my email because I am now 'following' the issue. I'm not gleefully just trying to hammer on this sw trying to find bugs to report as some perverse hobby :).

Anyway, enough of that, I love spyder and use it everyday. This beta release is so much better than 3.x and I'm happy to have access to it!

**Comment by ccordoba12** [comment]
@rlabbe, I understand your points but I think you're missing ours. The main point is that if you flood us with issues that diminishes our productivity instead of being helpful because we have to:

* See if the problem was already reported, if it's solved in master and if it can be reproduced.
* We usually have to ask for feedback and if we don't do it as soon as the issue was reported, most people forget about the problem and don't give us anything in return.
* We have to categorize the issue, assign it a milestone and a developer to solve it.

All those things take time (it could be 10 mins or half an hour per issue). So if you report more than 10 issues per day then that would take all my productive time, and I also have to review previous work and assign new one (because I'm the one who does issues triaging here).

I could leave your issues for later (as you suggest), but I don't want to have a huge backlog of issues to review in my Github notifications (because I also follow or I'm subscribed to a lot of other projects here) so most probably I'd start to ignore them.

Bottom line is this: if you want your issues to receive the attention you want, then please work with us and limit yourself in the amount of issues you want to open per day. Else we won't close them (I'm sure your issues are real problems because you do very good bug reports), but I don't think they'll be addressed as soon as you'd like to.
